src/__experimental__/components/checkbox/index.js

```diff
@@ -1 +1,5 @@
-export { default } from './checkbox.component';
```

Comment:
To keep everything in the same way I think that we can rewrite it to:
```
export { default, OriginalCheckbox } from './checkbox.component';
export { default as CheckboxGroup } from './checkbox-group.component';
```

Reply:
Updated in commit 7ac1fdf9a